spec/changelog.yml

```
@@ -4,461 +4,466 @@
 ##
 - version: 3.4.0-next
   changes:
-  - description: Add support for content packages.
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/777
+    - description: Add support for content packages.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/777
 - version: 3.3.0-next
   changes:
-  - description: Add support for `slo` assets.
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/767
+    - description: Add support for `slo` assets.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/767
+- version: 3.2.3-next
+  changes:
+    - description: Add support for identity of agentless deployment mode.
```

Nit. I would add that this identity is required, so it is more clear why this can be breaking.
```suggestion
    - description: Add support for identity of agentless deployment mode, and make it mandatory when this mode is enabled.
```